corehq/ex-submodules/casexml/apps/stock/consumption.py

```diff
@@ -138,7 +138,8 @@ def get_transactions(case_id, product_id, section_id, window_start, window_end):
     SimpleTransaction = collections.namedtuple('SimpleTransaction', ['action', 'value', 'received_on'])
 
     def _to_consumption_tx(txn):
-        if txn.type in (const.TRANSACTION_TYPE_STOCKONHAND, const.TRANSACTION_TYPE_STOCKOUT):
+        if txn.type in (const.TRANSACTION_TYPE_STOCKONHAND, const.TRANSACTION_TYPE_STOCKOUT,
+                        const.TRANSACTION_TYPE_LA):
```

Comment:
can you clarify how these are treated in the consumption calculation? I would have thought that they would be more similar to `consumption` than `stockonhand`